--- br_stp_if.c.orig	2014-02-28 19:38:25.296770400 +0900
+++ br_stp_if.c	2014-02-28 19:38:19.910645600 +0900
@@ -216,8 +216,12 @@
 	struct net_bridge_port *p;
 
 	/* user has chosen a value so keep it */
+	printk(KERN_INFO "%s: if (br->flags & BR_SET_MAC_ADDR) # br->flags (%ld) & BR_SET_MAC_ADDR (%d) \n",
+		br->dev->name, br->flags, BR_SET_MAC_ADDR);
 	if (br->flags & BR_SET_MAC_ADDR)
 		return;
+	printk(KERN_INFO "%s: mac address may be changed\n",
+		br->dev->name);
 
 	list_for_each_entry(p, &br->port_list, list) {
 		if (addr == br_mac_zero ||
